module/data/actor/templates/common.mjs

```diff
@@ -31,12 +31,37 @@ export default class CommonTemplate extends SystemDataModel.mixin(CurrencyTempla
           check: new FormulaField({required: true, label: "DND5E.AbilityCheckBonus"}),
           save: new FormulaField({required: true, label: "DND5E.SaveBonus"})
         }, {label: "DND5E.AbilityBonuses"})
-      }), {initialKeys: CONFIG.DND5E.abilities, label: "DND5E.Abilities"})
+      }), {
+        initialKeys: CONFIG.DND5E.abilities, initialValue: this._initialAbilityValue.bind(this),
+        prepareKeys: true, label: "DND5E.Abilities"
+      })
     });
   }
 
   /* -------------------------------------------- */
 
+  /**
+   * Populate the proper initial value for abilities.
+   * @param {string} key       Key for which the initial data will be created.
+   * @param {object} initial   The initial skill object created by SkillData.
+   * @param {object} existing  Any existing mapping data.
+   * @returns {object}         Initial ability object.
+   * @private
+   */
+  static _initialAbilityValue(key, initial, existing) {
+    const config = CONFIG.DND5E.abilities[key];
+    if ( config ) {
+      let defaultValue = config.defaults?.[this._systemType] ?? initial.value;
+      if ( typeof defaultValue === "string" ) defaultValue = existing[defaultValue]?.value ?? initial.value;
```

Comment:
Once we start having to cross-reference against sibling values, I feel like this leaves the realm of 'initialisation', and enters the realm of 'preparation'. It's true that we are simply trying to get the 'initial' value, but once things go beyond a simple config lookup, I wonder if it makes more sense to move these to preparation instead.

I think, since we're maintaining v10 compatibility, performing this as part of `MappingField#initialize` will have to do. But once we move to exclusively v11 compatibility, we should consider revisiting this and moving our `SystemDataModel` to extend from `TypeDataModel` instead, and provide a `prepareBaseData` implementation to perform this work.